packages/backend/src/controllers/bills/bills.controller.ts

```diff
@@ -0,0 +1,191 @@
+import {
+  Controller,
+  Post,
+  Get,
+  Put,
+  Param,
+  Delete,
+  Body,
+  HttpException,
+  HttpStatus,
+} from '@nestjs/common';
+import {
+  ApiCreatedResponse,
+  ApiOkResponse,
+  ApiNoContentResponse,
+  ApiOperation,
+  ApiTags,
+  ApiUnauthorizedResponse,
+} from '@nestjs/swagger';
+import { DecodedIdToken } from 'firebase-admin/auth';
+import { BillModel } from 'src/db/bill/bill.schema';
+import { BillStoreService } from 'src/db/bill/billStore.service';
+import { UserStoreService } from 'src/db/user/userStore.service';
+import { User } from 'src/util/user.decorator';
+import { CreateBillDto } from './dto/create-bill.dto';
+import { BillResponseDto, BillsResponseDto } from './dto/bill-response.dto';
+import { UpdateBillDto } from './dto/update-bill.dto';
+import { PayBillDto } from './dto/pay-bill.dto';
+import { BillUtil } from './bills.util';
+import { Auth } from '../../util/auth.decorator';
+
+@ApiTags('bills')
+@Controller('/api/v1/house/bills')
+@Auth()
+export class BillController {
+  constructor(
+    private readonly billStoreService: BillStoreService,
+    private readonly userStoreService: UserStoreService,
+    private readonly billUtil: BillUtil,
+  ) {}
+
+  @Get()
+  @ApiOperation({ summary: 'Retrieve all bills in the users flat.' })
+  @ApiOkResponse({
+    description: 'Bills retrieved successfully',
+    type: String,
+  })
+  async getBills(@User() user: DecodedIdToken): Promise<BillsResponseDto> {
+    const houseID = (await this.userStoreService.findOneByFirebaseId(user.uid))
+      .house;
```

Comment:
There should be a check for if the house document is undefined. Throw a bad request if it is